Dockerfile

```diff
@@ -28,5 +28,8 @@ RUN cd /pip && \
 	--no-deps -r requirements/docker.txt && \
     rm -r build cache
 
+# Need uglify for code minimization.
+RUN npm install
```

Comment:
`make update_npm_deps` should take care of this. The previous version of the Makefile did it automatically, since `initialize_docker` depended on `update_assets`, which depended on `update_npm_deps`.